src/hammer-vlsi/defaults_types.yml

```diff
@@ -0,0 +1,572 @@
+# Core configuration data types for the defaults for hammer-vlsi.
+# e.g. foo: str in this file means that the default type for foo is str.
+
+vlsi.core:
+  # Path(s) to out-of-tree hammer technology libraries.
+  technology_path: list[str]
+
+  # Technology to use. hammer-vlsi will read this and load the appropriate technology libraries.
+  technology: str
+
+  # Technology node dimension (nm) to use.
+  # Some tools change what they do as this changes.
+  node: str
+
+  # Build system to use, if any.
+  build_system: str
+
+  # Path(s) to synthesis tools.
+  # Add the built-in path to synthesis tools.
+  synthesis_tool_path: list[str]
+
+  # Path(s) to place and route tools.
+  # Add the built-in path to place and route tools.
+  par_tool_path: list[str]
+
+  # Path(s) to drc tools.
+  # Add the built-in path to drc tools.
+  drc_tool_path: list[str]
+
+  # Path(s) to lvs tools.
+  # Add the built-in path to lvs tools.
+  lvs_tool_path: list[str]
+
+  # Path(s) to sram generator tools.
+  # Add the built-in path to sram generator tools.
+  sram_generator_tool_path: list[str]
+
+  # Path(s) to sim tools.
+  # Add the built-in path to sim tools
+  sim_tool_path: list[str]
+
+  # Path(s) to power tools.
+  # Add the built-in path to power tools
+  power_tool_path: list[str]
+
+  # Path(s) to PCB deliverable tools.
+  # Add the built-in path to PCB deliverable tools.
+  pcb_tool_path: list[str]
+
+  # Synthesis tool to use.
+  synthesis_tool: str
+
+  # Place and route tool to use.
+  par_tool: str
+
+  # DRC tool to use.
+  drc_tool: str
+
+  # LVS tool to use.
+  lvs_tool: str
+
+  # SRAM Generator tool to use.
+  sram_generator_tool: str
+
+  # Sim tool to use.
+  sim_tool: str
+
+  # Power analysis tool to use.
+  power_tool: str
+
```

Comment:
Ack, it looks like some keys that I added in #651 aren't here because you started this before that was merged (e.g. `formal_tool` would be right here). Sorry - can you make one more run-through `defaults.yml` to get the new keys?